client/app/views/contact.coffee

```diff
@@ -61,10 +61,11 @@ module.exports = class ContactView extends ViewCollection
         @listenTo @model, 'remove', -> window.app.router.navigate '', true
 
     getRenderData: ->
-        _.extend {}, @model.toJSON(),
+        attrs = _.extend {}, @model.toJSON(),
```

Comment:
These are not actually the model attributes, if the purpose is to explicit this function return something, it would be better to just 
```return _.extend ...```